script/src/type_id.rs

```diff
@@ -72,6 +72,14 @@ impl<'a> TypeIdSystemScript<'a> {
                 LittleEndian::write_u64(&mut buf, input.since);
```

Comment:
Indeed, we don't need `block_hash` and `since`, `outpoint` of inputs already provides the uniqueness. It is also enough to use the first input out point.